src/drivers/cmakeDriver.ts

```diff
@@ -112,6 +112,10 @@ export abstract class CMakeDriver implements vscode.Disposable {
      */
     abstract get targets(): api.Target[];
 
+    public get targetsName(): string[] {
```

Comment:
Since `targets` is already available to use, I don't think we need to add this API.  You can just do it directly in the task provider.